src/core/Akka.Remote.Tests.MultiNode/RemoteNodeShutdownAndComesBackSpec.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.Remote.Transport;
+using Akka.TestKit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class RemoteNodeShutdownAndComesBackMultiNetSpec : MultiNodeConfig
+    {
+        public RemoteNodeShutdownAndComesBackMultiNetSpec()
+        {
+            First = Role("first");
+            Second = Role("second");
+
+            CommonConfig = DebugConfig(false).WithFallback(ConfigurationFactory.ParseString(@"
+                  akka.loglevel = INFO
+                  akka.remote.log-remote-lifecycle-events = INFO
+                  ## Keep it tight, otherwise reestablishing a connection takes too much time
+                  akka.remote.transport-failure-detector.heartbeat-interval = 1 s
+                  akka.remote.transport-failure-detector.acceptable-heartbeat-pause = 3 s
+                  ## the acceptable pause is too long and therefore the test will fail, it pass when we use a lower value like the default one
+                  ## akka.remote.watch-failure-detector.acceptable-heartbeat-pause = 60 s
```

Comment:
I took a look and I have to agree with you - I don't know why this value doesn't work with Akka.NET. I wonder if maybe it's an issue with the way the `watch-failure-detector` is implemented. I'll go and check that in a second.